recipes/elfutils/all/conanfile.py

```diff
@@ -156,14 +171,18 @@ def package_info(self):
         # library components
         self.cpp_info.components["libelf"].libs = ["elf"]
         self.cpp_info.components["libelf"].requires = ["zlib::zlib"]
-
-        self.cpp_info.components["libdw"].libs = ["dw"]
-        self.cpp_info.components["libdw"].requires = ["libelf", "zlib::zlib"]
         if self.options.with_bzlib:
-            self.cpp_info.components["libdw"].requires.append("bzip2::bzip2")
+            self.cpp_info.components["libelf"].requires.append("bzip2::bzip2")
         if self.options.with_lzma:
-            self.cpp_info.components["libdw"].requires.append("xz_utils::xz_utils")
-        if self.settings.os == "Linux":
+            self.cpp_info.components["libelf"].requires.append("xz_utils::xz_utils")
```

Comment:
@valgur were these changes a bug?

Reply:
No, this is the component that is actually directly linking against these libs.